voxel51/users/api.py

```diff
@@ -828,13 +828,15 @@ def is_job_expired(self, job_id=None, job=None):
             :class:`APIError` if the request was unsuccessful
         '''
         if job_id is not None:
-            expiration_date = self.get_job_details(job_id)["expiration_date"]
-        elif job is not None:
-            expiration_date = job["expiration_date"]
-        else:
+            return self.get_job_details(job_id)["expired"]
```

Comment:
This field is currently not returned (but can be)

Reply:
its useful. shall we return it??

Reply:
Yes, agreed that it's useful - ~but noting that this PR previously depended only on back-end changes that were already slated to be released, so it will be delayed a bit longer now~ actually, that's the JS PR. This will take a bit longer than #82, though.